frontend/src/data/config/MetricConfig.ts

```diff
@@ -408,4 +412,37 @@ export const METRIC_CONFIG: Record<string, VariableConfig[]> = {
       },
     },
   ],
+  vaccinated: [
+    {
+      variableId: "vaccinated",
+      variableDisplayName: "Vaccinated",
+      variableFullDisplayName: "Vaccinated Individuals",
```

Comment:
Site-wide it seems like we are inconsistent with when we use PEOPLE vs INDIVIDUALS. Is there a pattern, or is this something we should standardize?

Reply:
Good point, I just copied from "uninsured individuals", but we should talk about it and decide what we want.